### PR TITLE
vigra, bump to 1.11.1

### PR DESCRIPTION
**Pull request by Begasus**
Not tested on x86_64, vigra-config --cppflags and --libs give correct output, test still not working
**EDIT** checked the build on x86_64, builds ok :)

### CODE REVIEW

**Comment by Begasus** [comment]
Deleted the previous recipe as it's broken and doesn't serve any purpose (that I can find), builds ok for x86 (secondaryArch) and x86_64, vigra-config output is ok on both arch's, test still fail so I "?" all arch's, x86_gcc2 can't be build (from what I could find it needs a compiler > 4.4)

File: media-libs/vigra/vigra-1.11.1.recipe [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+By using template techniques similar to those in the C++ Standard Template \
+Library, you can easily adapt any VIGRA component to the needs of your \
+application, without thereby giving up execution speed."
+HOMEPAGE="http://ukoethe.github.io/vigra/"
+COPYRIGHT="1998-2017 by Ullrich Koethe"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/ukoethe/vigra/archive/Version-1-11-1.tar.gz"
+CHECKSUM_SHA256="b2718250d28baf1932fcbe8e30f7e4d146e751ad0e726e375a72a0cdb4e3250e"
+SOURCE_FILENAME="$portName-$portVersion.tar.gz"
```
Comment (fbrosson) [inline media-libs/vigra/vigra-1.11.1.recipe]:
```diff
-SOURCE_FILENAME="$portName-$portVersion.tar.gz"
+SOURCE_FILENAME="vigra-$portVersion.tar.gz"
```
(`$portName` = `vigra` when building on any primary arch...  and `vigra_x86` when building for x86 secondary arch.)

File: media-libs/vigra/vigra-1.11.1.recipe [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+SOURCE_DIR="vigra-Version-1-11-1"
```
Comment (fbrosson) [inline media-libs/vigra/vigra-1.11.1.recipe]: Suggestion:
```diff
-SOURCE_DIR="vigra-Version-1-11-1"
+SOURCE_DIR="vigra-Version-${portVersion//./-}"
```

File: media-libs/vigra/vigra-1.11.1.recipe [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+multi-dimensional images, because many algorithms (e.g. filters, feature \
+computation, superpixels) are implemented for arbitrary high dimensions. \
+By using template techniques similar to those in the C++ Standard Template \
+Library, you can easily adapt any VIGRA component to the needs of your \
+application, without thereby giving up execution speed."
+HOMEPAGE="http://ukoethe.github.io/vigra/"
+COPYRIGHT="1998-2017 by Ullrich Koethe"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/ukoethe/vigra/archive/Version-1-11-1.tar.gz"
```
Comment (fbrosson) [inline media-libs/vigra/vigra-1.11.1.recipe]: Suggestion:
```diff
-SOURCE_URI="https://github.com/ukoethe/vigra/archive/Version-1-11-1.tar.gz"
+SOURCE_URI="https://github.com/ukoethe/vigra/archive/Version-${portVersion//./-}.tar.gz"
```

File: media-libs/vigra/vigra-1.11.1.recipe (hunk 1 above)
Comment (Begasus) [reply media-libs/vigra/vigra-1.11.1.recipe]: Missed that one :) (left over from previous version, already removed a few instances for $portName there) :+1:

File: media-libs/vigra/vigra-1.11.1.recipe (hunk 2 above)
Comment (Begasus) [reply media-libs/vigra/vigra-1.11.1.recipe]: Thanks, I'll never get this right :)

**Pull request merged**